src/main/kotlin/nl/avisi/structurizr/site/generatr/site/SiteGenerator.kt

```diff
@@ -53,6 +54,12 @@ fun generateRedirectingIndexPage(exportDir: File, defaultBranch: String) {
     )
 }
 
+fun writeStructurizrJson(workspace: Workspace, exportDir: File){
+    val jsonFile = File(exportDir, "workspace.json")
```

Comment:
Suggestion: 

```kotlin
    val json = WorkspaceUtils.toJson(workspace, true)
    File(exportDir, "workspace.json").writeText(json)
```
Otherwise, LGTM!